### PR TITLE
Force knative-serving to master node

### PR DESCRIPTION
**Pull request by cvetkovic**
knative-serving placement moved to the master node which was previously empty.

### CODE REVIEW

File: scripts/cluster/create_multinode_cluster.sh [hunk 1]
```diff
@@ -42,6 +42,11 @@ mkdir -p $HOME/.kube
 sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
 sudo chown $(id -u):$(id -g) $HOME/.kube/config
 
+# enabling PodNodeSelector for control plane node affinity
```
Comment (ustiugov) [inline scripts/cluster/create_multinode_cluster.sh:45]: reason for this change?

File: scripts/cluster/setup_master_node.sh [hunk 2]
```diff
@@ -43,6 +43,12 @@ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.9.4/manife
 kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
 kubectl apply -f $ROOT/configs/metallb/metallb-configmap.yaml
 
+# export needed for envsubst
+export MASTER_NODE_NAME=$(kubectl get nodes -l node-role.kubernetes.io/master | tail -n 1 | cut -d ' ' -f 1)
+echo "Master node identified: ${MASTER_NODE_NAME}"
```
Comment (ustiugov) [inline scripts/cluster/setup_master_node.sh:48]:
```suggestion
echo "Master node: ${MASTER_NODE_NAME}"
```

**Review by ustiugov — CHANGES_REQUESTED** [review]
minor fixes requested

File: scripts/cluster/create_multinode_cluster.sh (hunk 1 above)
Comment (cvetkovic) [reply scripts/cluster/create_multinode_cluster.sh:45]: The old method of trying to fix this. Will be deleted.

**Review by ustiugov — APPROVED** [review]

**Comment by ustiugov** [comment]
feel free to rebase & merge

**Pull request merged**